src/main/java/ladder/view/InputView.java

```diff
@@ -0,0 +1,22 @@
+package ladder.view;
+
+import java.util.Scanner;
+
+public class InputView {
+    private static Scanner scanner;
+
+    static{
+        scanner = new Scanner(System.in);
+    }
```

Comment:
이 부분은 `private static Scanner scanner = new Scanner(System.in);` 하고 어떤 차이가 있나요?

Reply:
동일한것으로 알고있습니다. 저번에 로또할때 코드를 복붙하다보니..저땐 왜저렇게 했는지 의문이긴 하네요

Reply:
굳이 static {} 초기화 블록을 사용하신 이유가 있나요? 아래가 더 깔끔하지 않나 싶아서요
```suggestion
    private static final Scanner scanner = new Scanner(System.in);
```